Source/WebCore/page/DebugPageOverlays.cpp

```diff
@@ -498,7 +498,7 @@ void InteractionRegionOverlay::drawRect(PageOverlay&, GraphicsContext& context,
 
     auto region = activeRegion();
 
-    if (region || !valueForSetting("constrain"_s)) {
+    if ((region && region->type != InteractionRegion::Type::Occlusion) || !valueForSetting("constrain"_s)) {
```

Comment:
I think you want this check inside activeRegion() (i.e. never choose an occlusion region as active)